core/src/test/java/org/elasticsearch/common/settings/ScopedSettingsTests.java

```diff
@@ -319,7 +316,7 @@ public void testLoggingUpdates() {
             settings.applySettings(Settings.builder().put("logger._root", "TRACE").build());
             assertEquals("TRACE", ESLoggerFactory.getRootLogger().getLevel());
             settings.applySettings(Settings.builder().build());
-            assertEquals(level, ESLoggerFactory.getRootLogger().getLevel());
+            assertEquals(property != null ? property : level, ESLoggerFactory.getRootLogger().getLevel());
```

Comment:
That's just left over from when the test use to read in a system property that might be `null` but I later removed in favor of true randomization. I'll clean this up, `null` is not legit here.